tests/utils.py

```diff
@@ -169,13 +169,17 @@ def get_image_name_from_csv(image_string, csv_related_images):
     raise ResourceNotFoundError(f"no image with the string {image_string} was found in the csv_dict")
 
 
-def hotplug_resource_and_verify_hotplug(vm, client, sockets=None, memory_guest=None):
-    assert sockets or memory_guest, "No resource for update provided!!!"
+def hotplug_flat_vm_and_verify_hotplug(vm, client, sockets=None, memory_guest=None):
```

Comment:
```suggestion
def hotplug_spec_vm_and_verify_hotplug(vm, client, sockets=None, memory_guest=None):
```
just to be consistent in naming (since inside call is `hotplug_spec_vm`)